stdlib/public/core/VarArgs.swift

```diff
@@ -85,6 +85,30 @@ internal let _registerSaveWords = _countGPRegisters + _countSSERegisters * _sseR
 internal let _countGPRegisters = 16
 @_versioned
 internal let _registerSaveWords = _countGPRegisters
+
+#elseif arch(arm64)
+
+// ARM IHI 0055B
+// va_list may refer to any paramenter may be in one of three memory locations:
```

Comment:
Typo: parameter; missing word between “parameter” and “may”; needs wrapping to 80 characters